packages/ndla-ui/src/FileList/File.tsx

```diff
@@ -29,27 +34,64 @@ export interface FileFormat {
   tooltip: string;
 }
 
-const StyledFileItem = styled.li`
-  background: ${colors.brand.greyLighter};
-  display: flex;
-  align-items: center;
-  flex-wrap: wrap;
-  margin-bottom: ${spacing.xsmall};
-  padding: ${spacing.small};
-  ${mq.range({ from: breakpoints.tablet })} {
-    padding: ${spacing.small} ${spacing.normal};
-  }
-`;
-
-const File = ({ title, url, fileExists, fileType }: Props) => {
-  const { t } = useTranslation();
-  const tooltip = `${t("download")} ${url.split("/").pop()}`;
-
-  return (
-    <StyledFileItem>
-      <Format format={{ url, fileType, tooltip }} isPrimary title={title} isDeadLink={!fileExists} />
-    </StyledFileItem>
-  );
-};
+const StyledSafeLink = styled(SafeLink, {
+  base: {
+    fontWeight: "light",
+    color: "text.default",
+    padding: "unset",
+    minHeight: "unset",
+    paddingInline: "3xsmall",
+    textUnderlineOffset: "2px",
+    textDecoration: "underline",
+    _hover: {
+      textDecoration: "none",
+    },
+  },
+});
+
+const StyledHStack = styled(HStack, {
+  base: {
+    width: "100%",
+  },
+});
+
+const File = forwardRef<HTMLDivElement, FileProps>(
+  ({ title, url, fileExists, fileType, fileSize, children, ...rest }, ref) => {
+    const { t } = useTranslation();
+    const tooltip = `${t("download")} ${url.split("/").pop()}`;
+
+    return (
+      <StyledHStack justify="space-between" ref={ref} {...rest}>
+        <HStack gap="4xsmall">
```

Comment:
Hva er poenget med å ha to hstacker her?

Reply:
Fordi jeg tenkte det var smart å sende inn knapper inn i selve fil objektet 🥴  Det er det jo ikke!!!!!

Reply:
Du kan forenkle det til å bare vær en hstack.